docs/cedarling/getting-started/go.md

```
@@ -270,4 +386,10 @@ logs := instance.GetLogsByTag("info")
 
 ## Defined API
 
-Auto-generated documentation is available on [pkg.go.dev](https://pkg.go.dev/github.com/JanssenProject/jans/jans-cedarling/bindings/cedarling_go).
\ No newline at end of file
+Auto-generated documentation is available on [pkg.go.dev](https://pkg.go.dev/github.com/JanssenProject/jans/jans-cedarling/bindings/cedarling_go).
+
+## See Also
+
+- [Multi-Issuer Authorization Details](../cedarling-authz.md#multi-issuer-authorization-authorize_multi_issuer)
+- [JWT Mapping for Multi-Issuer](../cedarling-jwt-mapping.md#multi-issuer-jwt-mapping-authorize_multi_issuer)
+- [Policy Store Configuration](../cedarling-policy-store.md#multi-issuer-token-entities)
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)